CONTRIBUTORS.md

```diff
@@ -9,17 +9,62 @@ to make via an [issue](https://github.com/keylime/keylime/issues) or email to th
    outlining the fix or feature.
 2. Fork the keylime repository to your own github account and clone it locally.
 3. Hack on your changes.
-4. Update the README.md or documentation with details of changes to any
-   interface, this includes new environment variables, exposed ports, useful
-   file locations, CLI parameters and configuration values.
-5. Add and commit your changes with some descriptive text on the nature of the
-   change / feature in your commit message. Also reference the issue raised at
-   [1] as follows: `Fixes #45`. See [here](https://help.github.com/articles/closing-issues-using-keywords/)
-   for more message types
+4. Update the README.md or documentation in [keylime-docs](https://github.com/keylime/keylime-docs)
+   with details of changes to any interface, this includes new environment
+   variables, exposed ports, useful file locations, CLI parameters and
+   new or changed configuration values.
+5. Correctly format your commit message see [Commit Messages](#Commit Message Guidelines)
+   below.
 6. Ensure that CI passes, if it fails, fix the failures.
 7. Every pull request requires a review from the [core keylime team](https://github.com/orgs/keylime/teams/core)
    before merging.
 
+## Commit Message Guidelines
+
+We follow the commit formatting recommendations found [here](https://chris.beams.io/posts/git-commit/)
```

Comment:
Maybe replace "here" with something more descriptive: 

We follow the commit formatting recommendations found on Chris Beams' [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) article.

Reply:
done!

Reply:
Thanks!  Usually I try to avoid "click here" links for web accessibility reasons and people that need to use screen readers.